packages/wallets/src/components/AccountsList/AccountsList.tsx

```diff
@@ -1,8 +1,28 @@
 import React from 'react';
+import useDevice from '../../hooks/useDevice';
+import { TabList, TabPanel, TabPanels, Tabs } from '../Tabs';
 import { OptionsAndMultipliersListing } from '..';
 import './AccountsList.scss';
 
 const AccountsList = () => {
+    const { is_mobile } = useDevice();
+
+    if (is_mobile) {
+        return (
+            <Tabs className='wallets-accounts-list'>
+                <TabList list={['CFDs', 'Options & multipliers']} />
+                <TabPanels>
+                    <TabPanel>
+                        <h1>CFDs</h1>
```

Comment:
@adrienne-deriv let's left a localization needed comments here so we can fix them later on when we decided on this part